app/routes/events/components/EventEditor/index.tsx

```diff
@@ -127,20 +127,22 @@ const validate = createValidator({
 const EventEditor = () => {
   const params = useParams<{
     eventIdOrSlug: string;
-  }>();
+  }>() as {
+    eventIdOrSlug: string;
+  };
```

Comment:
Is this really how we have to do this now 💀

Reply:
Argh, yes... I don't know why they changed this to make all params optional, it worked fine in `react-router-v5`. And the params **are required**, there is no way to reach this path without specifying an event id...

I have a plan to make a custom hook like `useRequiredParams` or something, that just typecasts this a bit prettier.

Reply:
haha oh well 🤷🏼‍♂️